scripts/src/main/resources/scripts/command/az

```diff
@@ -1,106 +1,77 @@
 #!/usr/bin/env bash
-
-# autocompletion list
-if [ "${1}" = "shortlist" ]
-then
-  if [ -z "${2}" ]
-  then
-    echo "setup help"
-  fi
-  exit
-fi
-
-if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
 # shellcheck source=scripts/functions
 source "$(dirname "${0}")"/../functions
 
 function doSetup() {
-  if command -v az &> /dev/null
+  setLocation
+  if [ "${1}" != "silent" ] || [ ! -f "${AZ_FILE}" ] 
   then
-    if [ "${1}" != "silent" ] && ! doIsQuiet
+    if [ -d "${PYTHON_HOME}" ] 
```

Comment:
Actually you should even remove `setLocation` function and then do the check here as otherwise this is also not correct and we should have as little inter-dependencies between commandlets (here `az` and `python`).